### PR TITLE
Shield test gallery

### LINKED ISSUE #86: Wide shields showing up for 1 & 2 digit shields at DPR 1
When device pixel ratio (DPR) is > 1, standard width shields are used for 1 and 2 digit route numbers and the wide shield is used for 3 digit route numbers.  Here is DPR 2 for example:
DPR2

The same area at DPR 1 shows wide shields used for 1 and 2 digit route numbers which should be using the regular width shield:
DPR1

### PR DESCRIPTION
**Pull request by zekefarwell**
In order to more effeciently test out changes to shield rendering, this PR adds a bulk test gallery for shields at http://localhost:1776/shieldtest.html.  I extracted out the Overpass font and shield width logic adjustments to a separate PR (#544).  This way we can easily compare a shield test gallery with current text rendering vs any potential changes.

#### Shield test gallery in Mac Chrome, Safari, and Firefox
sans-serif-mac-chrome-safari-firefox

### CODE REVIEW

File: src/shieldtest.js
Comment (ZeLonewolf) [inline src/shieldtest.js:16]: Would it make sense to load these programmatically from the array that we generate in shield_defs.js?

File: src/shieldtest.js
Comment (ZeLonewolf) [inline src/shieldtest.js:191]: Should we choose ref values that represent the narrowest and widest possible characters?  So all `1`s for narrowest.  5 is a good choice for widest because it hits the corners.  Though refs with a W are probably the worst case.

File: src/shieldtest.js
Comment (zekefarwell) [reply src/shieldtest.js:16]: That was my initial plan and is what this line does:
https://github.com/ZeLonewolf/openstreetmap-americana/blob/903323c53e44796421fdf602d20d18dde05192d0/src/shieldtest.js#L188-L189

I replaced it with a custom list because it ends up including a lot of duplicate graphic since many networks values use the same shield:
Screen Shot 2022-08-22 at 11 15 05 PM

I left the commented out line there so we could toggle it back on when you want to see _all_ the networks, not just all the shields.

File: src/shieldtest.js
Comment (zekefarwell) [reply src/shieldtest.js:191]: That's what I attempted to do, but I may not have hit the mark exactly.  I'm imagining this purely as a developer tool for testing changes so this array could be modified at any time to test with different numbers.

File: src/shieldtest.js
Comment (zekefarwell) [reply src/shieldtest.js:16]: The full list also can take quite a long time to load and it crashed my browser tab at times.

Comment (ZeLonewolf) [reply src/shieldtest.js:16]: Ah yes, counties are dumb.

File: src/shieldtest.js
Comment (claysmalley) [inline src/shieldtest.js:175]: A bannered Ashland County shape.
```suggestion
  "US:OH:Business",
  "US:OH:ASD:TWP",
```

File: src/shieldtest.js
Comment (claysmalley) [inline src/shieldtest.js:180]: A bannered Texas shape.
```suggestion
  "US:TX:Park",
  "US:TX:FM:Business",
```

File: src/shieldtest.js
Comment (claysmalley) [inline src/shieldtest.js:182]: A bannered pill.
```suggestion
  "US:WI:Spur",
  "CA:ON:private_toll",
```

File: src/shieldtest.js
Comment (claysmalley) [inline src/shieldtest.js:62]: Non dimenticare gli ottagoni!
```suggestion
  "NL:S:Amsterdam",
  "IT:A-road",
```

File: src/shieldtest.js
Comment (claysmalley) [inline src/shieldtest.js:203]: Here are some extant 6-character refs, just for fun.
```suggestion
  "A 562",
  "1138-2",
  "A26/A7",
```

**Review by claysmalley — COMMENTED** [review]
I have a few more examples to add, just to be thorough.

File: src/shieldtest.js
Comment (claysmalley) [inline src/shieldtest.js:44]: Identical to `CA:AB:primary`.
```suggestion
```

File: src/shieldtest.js
Comment (zekefarwell) [reply src/shieldtest.js:24]: Sounds good.  I'm tried to get all the unique shape/color variants in the list, but I'm not surprised I missed some.

**Comment by zekefarwell** [comment]
This PR is now only the addition of the shield test gallery with the font and text rendering logic changes extracted to #544.  I think it should be good to merge this piece in unless anyone else requests changes.

**Review by claysmalley — APPROVED** [review]

**Review by ZeLonewolf — APPROVED** [review]
I just tested and it looks good to me.  As in #549, we should add documentation of this very cool debug feature, either in the contributor's guide or (better) integrate it into the existing samples button.

However, neither of these suggestions should hold up adding this capability.

**Comment by zekefarwell** [comment]
> we should add documentation of this very cool debug feature, either in the contributor's guide or (better) integrate it into the existing samples button.

I agree.  I've just added some documentation to the end of CONTRIBUTING.md.  

I looked into adding a link into the samples control, but it seemed like I'd need to do something hacky to make that work since the control is meant for OpenMapSamples and this is a separate thing.  I originally considered making this gallery as an OpenMapSample, but opted to go with this simpler option since the shields look exactly the same at all zoom levels and (at least for now) the shield rendering is very specific to this project only.

**Pull request merged**
